privaterelay/settings.py

```diff
@@ -230,7 +230,7 @@
         "emails.apps.EmailsConfig",
     ]
 
-if PHONES_ENABLED and TWILIO_ACCOUNT_SID and TWILIO_AUTH_TOKEN:
+if PHONES_ENABLED:
```

Comment:
Thanks, I'm much more comfortable with `PHONES_ENABLED` as the on/off switch, and leaving secrets out of it.